packages/forms-web-app/__tests__/unit/controllers/projects/documents/controller.component.test.js

```diff
@@ -0,0 +1,157 @@
+const {
+	getApplicationDocuments
+} = require('../../../../../src/controllers/projects/docments/controller');
+const { searchDocumentsV3 } = require('../../../../../src/services/document.service');
+const { getAppData } = require('../../../../../src/services/application.service');
+
+jest.mock('../../../../../src/services/application.service', () => ({
+	getAppData: jest.fn()
+}));
+jest.mock('../../../../../src/services/document.service', () => ({
+	searchDocumentsV3: jest.fn()
+}));
+describe('#getApplicationDocuments', () => {
+	describe('When getting the documents for the document library', () => {
```

Comment:
we need to add a test for the examination library states - 
`	if (parseInt(query.page) === 1 || !query || !query.page) {
		documents = await examinationLibrary(body, data.documents);
	} else {
		documents = data.documents;
	}
`

Reply:
done